lib/auth/methods.go

```diff
@@ -155,7 +155,12 @@ func (s *Server) AuthenticateUser(ctx context.Context, req AuthenticateUserReque
 		event.Code = events.UserLocalLoginCode
 		event.Status.Success = true
 
-		if err := s.CallLoginHooks(ctx); err != nil {
+		user, err := s.GetUser(actualUser, false)
```

Comment:
Could we push this to AuthenticateWeb/AuthenticateSSH, or return the queried user to them? Those methods will also query the user at the end.